src/url-params.ts

```diff
@@ -109,6 +146,17 @@ export class AnalyticsUrlParams {
 		return `${URL_PARAM_DEVICE_ID}=${encodeURIComponent(ids.join(','))}`;
 	}
 
+	/**
+	 * @return part of the session ID query parameter string that can be appended to URLs
+	 */
+	getSessionIdQueryString(): string {
+		const id = this.sessionId;
```

Comment:
Likewise, we need to update here to return `this.client.sessionId()` if it exists.